packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -151,94 +158,108 @@ const Trade = observer(() => {
         ),
         [open_market, try_synthetic_indices, try_open_markets]
     );
-
     const form_wrapper_class = is_mobile ? 'mobile-wrapper' : 'sidebar__container desktop-only';
     const chart_height_offset = React.useMemo(() => {
         if (is_accumulator) return '295px';
         if (is_turbos) return '300px';
         return '259px';
     }, [is_turbos, is_accumulator]);
 
+    const handleLaunchModal = () => {
+        setOpenLaunchModal(!open_launch_modal);
+        LocalStore.set('launchModalShown', true);
+    };
     return (
-        <div
-            id='trade_container'
-            className={classNames('trade-container', {
-                [`trade-container--${is_accumulator ? 'accumulators' : 'turbos'}`]: is_accumulator || is_turbos,
-            })}
-        >
-            <DesktopWrapper>
-                <PositionsDrawer />
-            </DesktopWrapper>
-            {/* Div100vhContainer is workaround for browsers on devices
-                    with toolbars covering screen height,
-                    using css vh is not returning correct screen height */}
-            <Div100vhContainer
-                id='chart_container'
-                className='chart-container'
-                is_disabled={isDesktop()}
-                height_offset={chart_height_offset}
+        <React.Fragment>
+            {open_launch_modal && is_logged_in && !is_already_shown && (
+                <LaunchModal isdarkMode={is_dark_theme} handleChange={handleLaunchModal} open={open_launch_modal} />
```

Comment:
`<LaunchModal/>` is waiting for `isDarkMode` prop and here it is `isdarkMode`. But I guess we mainly use snake_case in this project for variables

Reply:
changed to snake_case